modules/assim.sequential/R/downscale_function.R

```diff
@@ -1,33 +1,90 @@
+##' @title Preprocess Data for Downscaling
+##' @name NA_preprocess
+##' @author Sambhav Dixit
+##'
+##' @param data_path Character. File path for .rds containing ensemble data.
+##' @param coords_path Character. File path for .csv file containing the site coordinates, with columns named "lon" and "lat".
+##' @param date Character. If SDA site run, format is yyyy/mm/dd; if NEON, yyyy-mm-dd. Restricted to years within the file supplied to 'data_path'.
+##' @param C_pool Character. Carbon pool of interest. Name must match the carbon pool name found within the file supplied to 'data_path'.
```

Comment:
our style guide suggests all lower case; I'd prefer 'carbon_pool'

Reply:
is there a standard set of carbon pool names that are accepted by this function? if so, what are they; if not, should this function be written to support arbitrary names?

Perhaps ideally we would have a standard set of names for carbon pools, and we could separate out data-source specific standardization functions (analogous to model2netcdf) from the rest of the preprocessing?

Reply:
addressed issues in [ca14c09](https://github.com/PecanProject/pecan/pull/3321/commits/ca14c09f55aa088c741b23098e4c423492192f22) commit.